spec/integration/data_stream/manifest.spec.yml

```diff
@@ -126,6 +126,14 @@ spec:
               type: string
           required:
           - name
+        data_stream:
+          description: Data stream section of index template
+          type: object
+          additionalProperties: false
+          properties:
+            hidden:
+              description: Makes the data stream hidden
```

Comment:
Should it be remarked somehow the difference between the other `hidden` field? Or update the description of the other `hidden` field at the top level ?

Reply:
+1 to update the description and possibly deprecate the top level `hidden` field.